signingscript/docker.d/passwords.yml

```diff
@@ -369,8 +369,8 @@ in:
              ["autograph_gpg"]
           ]
           - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENIX_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
+             {"$eval": "AUTOGRAPH_FENNEC_USERNAME"},
+             {"$eval": "AUTOGRAPH_FENNEC_PASSWORD"},
```

Comment:
Nit: I think the environment variables should be `AUTOGRAPH_FENNEC_RELEASE_*` at least on the production instance:

https://github.com/mozilla-releng/scriptworker-scripts/blob/bcb921461799f180b8f2b4fb82237f03de6deef5/signingscript/docker.d/passwords.yml#L267-L274